word-guess.rb

```diff
@@ -0,0 +1,156 @@
+# This is a word guessing game (by Erika & Ari)
+# Guess the word before all the petals fall
+# use one class for game functionality
+require 'colorize'
+require 'spicy-proton'
+
+
+class Game
+
+ def initialize(word)
+   @correct_word_array = word.downcase.split("")
+   # This take the random word, downcase, and split letters into it's own indexes
+   # or elements (interchangable right?)
+   @correct_guess = word.gsub(/./ , "_").split("")
+# gsub method lets us sub in "-". "/./" is for any characters.
+# split method puts "-" into own index or makes them their own elements
+   @no_match_guesses = []
+   @failed_attempt = 0
+ end
+
+ def pedal_remove
+   pedal_remove = [
+     '''
+       (@)(@)(@)(@)(@)
+          ,\,\,|,/,/,
+            _\|/_
+           |_____|
+            |   |
+            |___|'''.cyan,'''
+
+        (@)(@)(@)(@)
+        ,\,\,|,/,/,
+            _\|/_
+           |_____|
+            |   |
+            |___|'''.green,'''
+
+          (@)(@)(@)
+         ,\,\,|,/,/,
+            _\|/_
+           |_____|
+            |   |
+            |___|'''.yellow,'''
+
+           (@)(@)
+         ,\,\,|,/,/,
+            _\|/_
+           |_____|
+            |   |
+            |___|'''.blue,'''
+
+              (@)
+          ,\,\,|,/,/,
+             _\|/_
+            |_____|
+             |   |
+             |___|'''.red]
```

Comment:
I suspect this is where your indentation gets messed up - Atom will want you to end the array on a new line.

```ruby
             |___|'''.red
    ]
  end
```